docs.openc3.com/docs/getting-started/key_concepts.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 4
+title: Key Concepts
+---
+
+# OpenC3 COSMOS Key Concepts
+
+## Projects
+
+The main COSMOS [repo](https://github.com/OpenC3/cosmos) contains all the source code used to build and run COSMOS. However, users (not developers) of COSMOS should use the COSMOS [project](https://github.com/OpenC3/cosmos-project) to launch COSMOS. The project consists of the [openc3.sh](https://github.com/OpenC3/cosmos-project/blob/main/openc3.sh) and [openc3.bat](https://github.com/OpenC3/cosmos-project/blob/main/openc3.bat) files for starting and stopping COSMOS, the [compose.yaml](https://github.com/OpenC3/cosmos-project/blob/main/compose.yaml) for configuring the COSMOS containers, and the [.env](https://github.com/OpenC3/cosmos-project/blob/main/.env) file for setting runtime variables.
```

Comment:
Additionally it contains user modifiable config files for both Redis and Traefik.